<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Сортировки сортировочки</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <widget class="QTextEdit" name="tbInput">
        <property name="font">
         <font>
          <family>Cascadia Code SemiBold</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QTextEdit" name="tbOutput">
        <property name="font">
         <font>
          <family>Cascadia Code SemiBold</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>774</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="btnLoad"/>
    <addaction name="btnSave"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Отсортировать</string>
    </property>
    <addaction name="btnBubble"/>
    <addaction name="btnSelection"/>
    <addaction name="btnQuick"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="btnLoad">
   <property name="text">
    <string>Загрузить</string>
   </property>
  </action>
  <action name="btnSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="btnBubble">
   <property name="text">
    <string>Пузырьком</string>
   </property>
  </action>
  <action name="btnSelection">
   <property name="text">
    <string>Выбором</string>
   </property>
  </action>
  <action name="btnQuick">
   <property name="text">
    <string>Быстро</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
